server/testll.c

```diff
@@ -0,0 +1,46 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/queue.h>
+
+typedef struct thread_data
```

Comment:
@neohere97 nice idea to add a test case for this.  You could incorporate into the unity test framework using the existing test examples at https://github.com/cu-ecen-aeld/aesd-assignments/blob/master/student-test/assignment1/Test_validate_username.c

Reply:
Yes! I'll look into it